### ISSUE TITLE
Possibility of using import syntax on server in production mode.

### ISSUE THREAD

**Issue by joeyshipley** [brillout/vite-plugin-ssr#118]
Exploring the possibility of using 'import' syntax instead of 'require' on server in production mode.

Example Repo: https://github.com/joeyshipley/vite-plugin-ssr-server-import-syntax

In package.json
```
{
  "engines": {
    "node": "16.x"
  },
  "type": "module",
```

In server.js
```
import path from 'path';
import { fileURLToPath } from 'url';
import express from 'express';
import vite from 'vite';
import { createPageRender } from 'vite-plugin-ssr';
```

**Comment by brillout** [comment]
Perfect, I'll have a look at it tmrw

**Comment by brillout** [comment]
Fix released in `0.2.2`.
